pubspec.yaml

```diff
@@ -167,7 +174,7 @@ flutter_intl:
   # Because it will be repeatedly generated when it is turned on,
   # it is very troublesome, so we temporarily turn it off.
   # If necessary, please turn it on by yourself.
-  enabled: false
+  enabled: true
```

Comment:
Please revert this change since we don't expect it to be set to `true`.

Reply:
resolve commit : [6f5fdf4](https://github.com/NEO-TAT/tat_flutter/pull/83/commits/6f5fdf4f1d2bfd694669e5e1bb855530e6c6ba2b)